### ISSUE TITLE
More huge Embur wart variants to grow from the small wart

### ISSUE THREAD

**Issue by saltyseadoggo** [CorgiTaco/BYG#387]
**Is your feature request related to a problem? Please describe.**
The huge Embur warts grown from using bone meal on an Embur wart are very repetitive. I grew eleven of them in a testing world, and all of them were one of two different variants. Growing the same 'tree' over and over feels strange and not alligned with vanilla gameplay, where trees grown by the player have subtle leaf variations, different shapes in the case of the acacia, different branches in the case of the dark oak...

**Describe the solution you'd like**
I would like to see some more variants of the huge Embur wart that can grow from its 'sapling'/wart to break up the repetition!
As a side note, I would also like to see at least some of them use the Embur gel vines, so that they can be obtained from growing the warts manually.

**Additional context**
My and my partner's *All of Fabric 3* testing world, where I grew the warts and realised this. Every wart here is one of two variants. The third image is from Xaero's map mod; note that all eleven warts are one of two shapes.
![2021-04-04_19 29 18](https://user-images.githubusercontent.com/69029346/113531181-ea0aed00-957c-11eb-962b-72e82712cbfc.png)
![2021-04-04_19 29 27](https://user-images.githubusercontent.com/69029346/113531184-eb3c1a00-957c-11eb-80fd-74e35f790571.png)
![2021-04-04_19 29 51](https://user-images.githubusercontent.com/69029346/113531185-ebd4b080-957c-11eb-9216-f30c9f46c78c.png)

**Comment by CorgiTaco** [comment]
We only have 2 variants of the Embur Mushroom at the moment. If you'd like, you can submit Embur Mushroom NBT structures here and we'll be happy to consider them(there's no guarantees)

Sorry :(

**Comment by saltyseadoggo** [comment]
I took a stab and built some Embur mushroom variants. They're all on the smaller side compared to the current ones, which I intended as a bit of variety, and they all use either the current cap shapes or a smaller variant that I made. They all also include Embur gel vines, which I felt were underutilized in the biome.
![2021-05-04_22 56 37](https://user-images.githubusercontent.com/69029346/117102000-3a1bd180-ad2c-11eb-8982-faaba95b0336.png)
[embur_warts.zip](https://github.com/CorgiTaco/BYG/files/6425245/embur_warts.zip)
